src/robot_dart/graphics/camera_osr.hpp

```diff
@@ -27,14 +29,15 @@ namespace robot_dart {
                 width = vp->width();
                 height = vp->height();
 
-                _viewer->image()->readPixels(x, y, width, height, GL_RGB, GL_UNSIGNED_BYTE);
+                _viewer->image()->readPixels(x, y, width, height, GL_LUMINANCE, GL_UNSIGNED_BYTE);
```

Comment:
Why did you change this to `GL_LUMINANCE`? In my computer, I am getting very sharp images, like this one:

![cam2](https://user-images.githubusercontent.com/1283922/60283025-992a8700-9908-11e9-8fd7-8051ee914dcc.png)

Reply:
I will revert this, it was just easier to do some tests with it as it forms an image with a single channel (the red one)